### DISCUSSION TITLE
Why does lighthouse cli not seem to use the VPN that local browsers do

### DISCUSSION THREAD

**Discussion by iDVB** [GoogleChrome/lighthouse#14406] [Q&A]
When I open a dev url (behind IP Whitelist) in the chrome browser directly (while connected to company VPN - OpenVPN) I can see the site just fine.

However when I run a lighthouse cli…. it seems to fail with 403, as if not benefiting from the VPN.

**Comment by brendankenny** [comment]
I haven't used OpenVPN in a long while, but does it still use a Chrome extension by any chance? Lots of things are turned off in the Chrome launched by the CLI to try to minimize background browser activity and anything that might interfere with page content/performance, and extensions are one of the things disabled.

You can see the current list of [Chrome flags disabling things](https://github.com/GoogleChrome/chrome-launcher/blob/f618e7e5dd96ff30805f5ece92a27eacfd2d41c4/src/flags.ts#L10-L50) and a [handy reference for what each flag does](https://github.com/GoogleChrome/chrome-launcher/blob/master/docs/chrome-flags-for-tools.md) over in `chrome-launcher`. If the problem isn't a disabled extension, something else in there could be blocking whatever hooks OpenVPN relies on (proxy settings or corp management or whatever).

You can try `--ignore-default-chrome-flags` to disable _all_ those flags and see if the VPN works in that case, though that comes with the tradeoff of things like a first-run page, background syncing on startup, etc. Unfortunately there's not a great way to get some but not all the flags disabled. You'd need to do something like `--ignore-default-chrome-flags --chrome-flags="--disable-features=Translate --disable-extensions --disable-component-extensions-with-background-pages --disable-background-networking ..."`, copying from that chrome-launcher file, but with the offending flag(s) removed.
